packages/components/src/components/search/_search.scss

```diff
@@ -276,6 +299,43 @@
       color: transparent;
     }
   }
+
+  .#{$prefix}--search--expandable .#{$prefix}--search-input {
+    width: 0;
+    padding: 0;
+    transition-timing-function: motion(standard, productive);
+    transition-duration: $duration--fast-01;
+    transition-property: width, padding;
```

Comment:
Cool, so it looks like this will inherently support animating leftwards, rightwards, or both from the center 👍